configure.py

```diff
@@ -877,6 +889,11 @@ def _fribidi_install(cfg):
             _harfbuzz_install,
             _fribidi_install,
         ],
+        "Darwin": [
+            _nopemd_install,
+            _glslang_install,
+            _moltenvk_install,
+        ],
```

Comment:
MoltenVK needs to be removed from `doc/usr/howto/installation.md`

Reply:
Done.